rpmlint/version.py

```diff
@@ -1,6 +1,6 @@
 try:
-    from importlib.metadata import version as implib_metadata_version
+    from importlib.metadata import version
 except ImportError:
-    from importlib_metadata import version as implib_metadata_version
+    from importlib_metadata import version
 
-__version__ = implib_metadata_version('rpmlint')
+__version__ = version(__name__)
```

Comment:
Is it safe to use the `version()` name like this? And where does `__name__` come from?

Reply:
Nope, i was just trying if it would resolve it.